src/theme.js

```diff
@@ -8,13 +8,16 @@ export const theme = {
     lightGrey: "#F5F5FA",
     white: "#FFFFFF",
     blue: "#0044CC",
+    blueRibbon: "#004de6",
+    lighterBlue: "#0055ff",
     lightBlue: "#D6E4FF",
     yellow: "#FCD420",
   },
 
   breakpoints: {
     small: "480px",
     large: "992px",
+    largest: "1380px",
     mobileMin: 320,
     mobileMax: 767,
```

Comment:
Wydaje mi się że nikt już na tym nie pracuje, więc można to usunąć ;D

Reply:
ok :)

Reply:
Ok

Reply:
bede wdzieczna zeby zostaly, jezeli nie przeszkadzaja, ja glownie na tym bazowalam nad responsywnoscia, ale i tak bede musiala dopracowac, takze jestem za tym, zeby nie usuwac :)

Reply:
Dobrze, zostawię waszą wersje ;D